docs/Release_Guide/metviewer_official.rst

```diff
@@ -8,4 +8,14 @@ Create a new vX.Y.Z official release from the develop branch.
 .. include:: release_steps/open_release_issue.rst
 .. include:: release_steps/clone_project_repository.rst
 .. include:: release_steps/checkout_develop_branch.rst
-.. include:: release_steps/create_release_feature_branch.rst
+.. include:: release_steps/metviewer/update_version_official.rst
```

Comment:
Above this line ".. include:: release_steps/metviewer/update_version_official.rst" should be .. include:: release_steps/create_release_feature_branch.rst
(Purpose is to update the version number and add release notes and have those reviewed in a pull request and is referred to in release_steps/merge_release_issue.rst which is included already)